debezium-bom/pom.xml

```diff
@@ -575,17 +576,22 @@
             <dependency>
                 <groupId>dev.langchain4j</groupId>
                 <artifactId>langchain4j-embeddings-all-minilm-l6-v2</artifactId>
-                <version>${version.langchain4j}</version>
+                <version>${version.langchain4j.unstable}</version>
```

Comment:
@vjuranek Do we need to have it marked as `unstable`? We don't use such approach elswhere.

Reply:
I'll take a look if it can be workarounded somehow (e.g. importing bom file), but this seems to the approach langchain4j decided to go - until recently all artifacts have the same version, e.g. `1.0.0-beta2`. recently, they did first stable release, but only part of the artifacts (e.g. `core`) are considered stable. These artifacts have version wihout `-beta` suffix, e.g. `1.0.1`, while artifacts which are not considered stable has this suffix, e.g. `1.0.1-beta6`. I don't know if they plan different release cadence for stable/unstable artifacts, or what is the advantage here, but they use two different versions for the artifacts in the same release.

Reply:
well, switching to bom file seems to fix the issue